subsys/net/l2/openthread/openthread.c

```diff
@@ -22,6 +22,7 @@ LOG_MODULE_REGISTER(net_l2_openthread, CONFIG_OPENTHREAD_L2_LOG_LEVEL);
 #include <openthread/cli.h>
```

Comment:
>Also initialisation for raw link packet interface has been added. Can be
used for picking up ZigBee frames and interpreting them in the
application.

Where is it? What is a _ZigBee frame_? Is OpenThread not based on IEEE802.15.4?

Reply:
It is part of upstream openthread API. using it here:
https://github.com/zephyrproject-rtos/zephyr/pull/23249/files#diff-bf3f4a47fa194d5e0de369c2cfe68f1aR214-R217
https://github.com/zephyrproject-rtos/zephyr/pull/23249/files#diff-b1484649f09207a98f1953e0ef947964R413-R414

The mapping to https://github.com/zephyrproject-rtos/openthread/blob/zephyr/etc/cmake/options.cmake#L124-L127 is done in https://github.com/zephyrproject-rtos/openthread/pull/6/files#diff-83aeef700a5a192d68e562d9ccfe3725R36-R38.

Using the API you can get access to all IEEE802.15.4, also non-Thread packets like ZigBee packets.

Reply:
Than please fix your commit and replace ZigBee with IEEE802.15.4.

Reply:
Replaced ZigBee with 802.15.4 in the commit message. Anything else I should change?